test/methods.js

```diff
@@ -400,6 +400,32 @@ asyncTest("remote", function() {
 	strictEqual( v.element(e), true, "still invalid, because remote validation must block until it returns; dependency-mismatch considered as valid though" );
 });
 
+asyncTest("remote, pending class added to element while call outstanding", function() {
+	expect(3);
+	var e = $("#username"),
+		v = $("#userForm").validate({
+			rules: {
+				username: {
+					remote: "users.php"
+				}
+			},
+			submitHandler: function() {
+				ok( false, "submitHandler may never be called when validating only elements");
+			}
+		});
+
+	$(document).ajaxStop(function() {
```

Comment:
success callback didn't work, because it gets invoked before the pending class is removed. However, the complete callback worked fine.